web-application/mini-project/script.js

```diff
@@ -0,0 +1,84 @@
+const app = Vue.createApp({
+    data() {
+        return {
+            postData: {
+                message: '',
+                imageUrl: null,
+                location: null,

```

Comment:
Some improvements to displaying the location
- Display the lat and long details obtained from HTML5 and display it under the post
- Try to integrate Google Maps SDK and show a mini-map of the post location by passing the lat, long details and display it under the post.